manim/mobject/value_tracker.py

```diff
@@ -92,41 +92,83 @@ def __bool__(self):
         """Return whether the value of this value tracker evaluates as true."""
         return bool(self.get_value())
 
+    def __add__(self, value: Union[int, float]):
```

Comment:
```suggestion
    def __add__(self, value: float):
```

Reply:
We've previously discussed and agreed that just using `float` as the typehint is fine.